charts/lotus-gateway/templates/statefulset-gateway.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Could this be a `Deployment` instead of `Statefulset` ?

Reply:
Yep, switching.